src/pveclib/vec_int32_ppc.h

```diff
@@ -536,6 +543,60 @@ vec_clzw (vui32_t vra)
   return ((vui32_t) r);
 }
 
+/** \brief Vector Count Trailing Zeros word.
+ *
+ *  Count the number of trailing '0' bits (0-32) within each word
+ *  element of a 128-bit vector.
+ *
+ *  For POWER9 (PowerISA 3.0B) or later use the Vector Count Trailing
+ *  Zeros Word instruction <B>vctzw</B>. Otherwise use a sequence of
+ *  pre ISA 3.0 VMX instructions.
+ *  SIMDized count Trailing zeros inspired by:
+ *
+ *  Warren, Henry S. Jr and <I>Hacker's Delight</I>, 2nd Edition,
+ *  Addison Wesley, 2013. Chapter 5 Counting Bits, Section 5-4.
+ *
+ *  |processor|Latency|Throughput|
+ *  |--------:|:-----:|:---------|
+ *  |power8   |  6-8  | 2/cycle  |
+ *  |power9   |   3   | 2/cycle  |
+ *
+ *  @param vra 128-bit vector treated as 4 x 32-bit integer (words)
+ *  elements.
+ *  @return 128-bit vector with the Trailng Zeros count for each word
+ *  element.
+ */
+static inline vui32_t
+vec_ctzw (vui32_t vra)
+{
+  vui32_t r;
+#ifdef _ARCH_PWR9
+#if defined (vec_cnttz)
+  r = vec_cnttz (vra);
+#elif defined (__clang__)
+  r = vec_cnttz (vra);
+#else
+  __asm__(
+      "vctzw %0,%1;"
+      : "=v" (r)
+      : "v" (vra)
+      : );
+#endif
+#else
+// For _ARCH_PWR8 and earlier. Generate 1's for the trailing zeros
+// and 0's otherwise. Then count (popcnt) the 1's. _ARCH_PWR8 uses
+// the hardware vpopcntw instruction. _ARCH_PWR7 and earlier use the
+// PVECLIB vec_popcntw implementation which runs ~20-28 instructions.
```

Comment:
This comment about the implementation of vec_popcntw makes more sense there, close to the implementation.

Reply:
Oh, it already has a similar comment:
https://github.com/open-power-sdk/pveclib/blob/5901c9cc5b9f836ec87d12a13b127bc185097699/src/pveclib/vec_int32_ppc.h#L1077

Cheers! :)

Reply:
I assume this is ok (not changes required)?

Reply:
Yes! :)